website/static/_redirects

```diff
@@ -0,0 +1,880 @@
+
+## refocus deploy page
+/docs/collaborate/environments/environments-in-dbt /docs/environments-in-dbt 301
```

Comment:
i think our new redirects are now in https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/vercel.json since the redirects file doesn't exist anymore